webapp-django/crashstats/settings/base.py

```diff
@@ -682,3 +682,11 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 
 CRASH_ANALYSIS_URL = 'https://crash-analysis.mozilla.com/crash_analysis/'
+
+# At what point do we consider crontabber to be stale.
+# Ie. if it hasn't run for a certain number of hours we'd consider
+# that a failing situation and it'll trigger monitoring.
+CRONTABBER_STALE_HOURS = config(
+    'CRONTABBER_STALE_HOURS',
+    default=24
```

Comment:
That seems very high to me as a default. Shouldn't anyone worry a lot sooner than 24 hours after crontabber started stalling? Or is it because it's possible to configure crontabber to run only once per day in some installs?